### DISCUSSION TITLE
Popover Mobile / Desktop - Bug

### DISCUSSION THREAD

**Discussion by BrunoBiagioniNeto** [Telefonica/mistica-design#1506] [Proposals]
### Goal

Componente não está desligando o Asset no Figma conforme imagem.

Também não temos a opção de ligar e desligar as setas.

![image](https://github.com/Telefonica/mistica-design/assets/123083886/ac52f409-2e49-4f09-a1cf-1390471041f1)

### Context

Quando desligamos todas opções de icone, asset e slot o container continua visível no figma.

### Impact

Designers não conseguem usar o componente corretamente.

### Discussion Questions

- Figma requer atualização.

**Comment by yceballost** [comment]
Lo convierto en issue directamente! gracias por el report @BrunoBiagioniNeto!
